core/scss/components/sidebar/sidebar.scss

```diff
@@ -49,6 +52,20 @@
     &--minimized {
       width: rem-calc(68);
       min-width: 0;
+
+      .md-sidebar__avatar {
```

Comment:
Did we talk about changing these back to avatar specific?

Reply:
taking this out, shouldn't have any avatar in the sidebar styles